src/components/weather/QuickForecastCard.vue

```diff
@@ -56,19 +56,30 @@
 <script>
 export default {
   name: "QuickForecastCard",
+
+  data: () => ({
+    message: null,
+    loading: false,
+  }),
   computed: {
     isLoading() {
       return this.loading === true;
     },
   },
-  data: () => ({
-    message: null,
-    loading: false,
-  }),
+
   methods: {
-    getPosition() {
-      this.message = "Geolocation";
+    getPosition: function () {
+      navigator.geolocation.getCurrentPosition(this.showPosition, this.error);
     },
+    showPosition: function (position) {
+      this.lat = position.coords.latitude;
+      this.lon = position.coords.longitude;
+      this.message = "Geolocation:{" + this.lat + "," + this.lon + "}";
+    },
+    error: function () {
+      this.$alert("Geolocation error");
```

Comment:
Si riesce a dare qualche informazione in più all'utente? Ad esempio, se io all'inizio ho negato il permesso di accedere alla mia posizione, poi posso comunicare in questo alert, quando è ora, che l'utente mi deve dare il permesso?

Reply:
Certo, posso esprimere diversi errori. ora lo faccio